bookbotproject/conversationtree/models.py

```diff
@@ -3,9 +3,24 @@
 
 from django.db import models
 
+class ConversationNode(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    #Every node has a title to describe their purpose.
+    title = models.CharField(max_length=100, blank=True)
+    # Every node has an intent. (Will be used to choose which child the parent will select to go).
+    intent = models.CharField(max_length=50, blank=True)
```

Comment:
intent field may not satisfy our needs. We can make it a longer CharField, like 100

Reply:
I thought the intents to be shorter but you are right, lets make it 100 chars.